crates/red_knot_python_semantic/resources/mdtest/dataclasses.md

```diff
@@ -560,12 +560,10 @@ class ConvertToLength:
 class C:
     converter: ConvertToLength = ConvertToLength()
 
-# TODO: Should be `(converter: str = Literal[""]) -> None` once we understand overloads
-reveal_type(C.__init__)  # revealed: (converter: str = str | int) -> None
+reveal_type(C.__init__)  # revealed: (converter: str = str) -> None
```

Comment:
I think this should be `str` instead of `Literal[""]` looking at the overloaded definition of `ConvertToLength.__get__` above (cc @sharkdp)

Reply:
Yes, you're right. Can you please change the overload above to the following?
```
def __get__(self, instance: None, owner: type) -> Literal[""]: ...
```